openwisp_utils/admin_theme/templates/admin/base_site.html

```diff
@@ -65,7 +65,7 @@ <h1 id="site-name"><a href="{% url 'admin:index' %}">{{ site_header }}</a></h1>
     if we put it in extrahead it will be
     loaded before jQuery is loaded
 {% endcomment %}
-{% if has_permission %}
+{% if user.is_authenticated %}
```

Comment:
I think we should check whether `user.is_staff` as well, otherwise the menu is shown also in non admin templates.

Reply:
You are right!

Reply:
Let's leave it now, I created a new issue for this: https://github.com/openwisp/openwisp-utils/issues/236